manifests/rule/local.pp

```diff
@@ -154,29 +154,30 @@
   Boolean                        $sync                                 = false,
   Optional[String[1]]            $sig_provider                         = undef,
   Optional[String[1]]            $cry_provider                         = undef,
+  Simplib::PuppetLogLevel        $queue_validation_log_level           = simplib::dlookup('rsyslog::rule::local', 'queue_validation_log_level', $name, { 'default_value' => 'warning' }),
   Optional[Stdlib::Absolutepath] $queue_filename                       = undef,
   Optional[Stdlib::Absolutepath] $queue_spool_directory                = undef,
   Optional[Integer[0]]           $queue_size                           = undef,
-  Integer[0]                     $queue_dequeue_batch_size             = 16,
+  Optional[Integer[0]]           $queue_dequeue_batch_size             = undef,
   Optional[Integer[0]]           $queue_max_disk_space                 = undef,
   Optional[Integer[0]]           $queue_high_watermark                 = undef,
-  Integer[0]                     $queue_low_watermark                  = 2000,
+  Optional[Integer[0]]           $queue_low_watermark                  = undef,
   Optional[Integer[0]]           $queue_full_delay_mark                = undef,
   Optional[Integer[0]]           $queue_light_delay_mark               = undef,
-  Integer[0]                     $queue_discard_mark                   = 9750,
-  Integer[0]                     $queue_discard_severity               = 8,
+  Optional[Integer[0]]           $queue_discard_mark                   = undef,
+  Optional[Integer[0]]           $queue_discard_severity               = undef,
   Optional[Integer[0]]           $queue_checkpoint_interval            = undef,
   Boolean                        $queue_sync_queue_files               = false,
   Rsyslog::QueueType             $queue_type                           = 'Direct',
-  Integer[0]                     $queue_worker_threads                 = 1,
-  Integer[0]                     $queue_timeout_shutdown               = 0,
-  Integer[0]                     $queue_timeout_action_completion      = 1000,
-  Integer[0]                     $queue_timeout_enqueue                = 2000,
-  Integer[0]                     $queue_timeout_worker_thread_shutdown = 60000,
-  Integer[0]                     $queue_worker_thread_minimum_messages = 100,
-  String[1]                      $queue_max_file_size                  = simplib::dlookup('rsyslog::rule::local', 'queue_max_file_size', $name, { 'default_value' => '1m' }),
```

Comment:
We should leave the `dlookup` here. Should be able to still make it `Optional` and return `undef` by default.

Reply:
Bah, I missed the update to metadata.json. We should leave the `dlookup` though

Reply:
Fixed.